package.json

```diff
@@ -110,7 +110,7 @@
     "@luma.gl/core": "^8.2.0",
     "@luma.gl/shadertools": "^8.2.0",
     "fast-xml-parser": "^3.16.0",
-    "geotiff": "ilan-gold/geotiff.js#ilan-gold/string_fix",
+    "geotiff": "ilan-gold/geotiff.js#ilan-gold/viv_050_release",
```

Comment:
This is the branch I was referring to - https://github.com/ilan-gold/geotiff.js/tree/ilan-gold/viv_050_release

It has both `AbortController` and the async-ify + LZW changes.